test/extended/apiserver/resource_quota.go

```diff
@@ -0,0 +1,60 @@
+package apiserver
+
+import (
+	"context"
+
+	g "github.com/onsi/ginkgo/v2"
+	o "github.com/onsi/gomega"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("[sig-api-machinery][Feature:APIServer][Serial]", func() {
+	defer g.GinkgoRecover()
+
+	oc := exutil.NewCLI("apiserver-resource-quota")
+
+	g.It("APIServer The number of created persistent volume claims can not exceed the limitation", func() {
```

Comment:
APIServer is redundant here. We have `[Feature:APIServer]` in case title.

Reply:
Updated